ui/app/src/utils/content.ts

```diff
@@ -466,13 +466,23 @@ function parseElementByContentType(
             // TODO: Groovy Controller Issue;
             path = getInnerHtml(item.querySelector(':scope > key'));
           }
-          const instance = parseContentXML(
-            component ? wrapElementInAuxDocument(component) : null,
-            path,
-            contentTypesLookup,
-            instanceLookup
-          );
-          array.push(instance);
+          const itemKey = getInnerHtml(item.querySelector(':scope > key'));
+          // Embedded components have no .xml in their key
+          const isFile = Boolean(itemKey) && !itemKey.endsWith('.xml') && Boolean(!item.getAttribute('inline'));
```

Comment:
@russdanner @sumerjabri FYI this is the kind of juggling we have to do because of the mixed datasource (item selector used for files, assets, code, AND content)